components/eam/src/physics/cam/shoc_intr.F90

```diff
@@ -408,6 +408,9 @@ subroutine shoc_init_e3sm(pbuf2d, dp1_in)
     call addfld('CONCLD',(/'lev'/),  'A',        'fraction', 'Convective cloud cover')
     call addfld('BRUNT',(/'lev'/), 'A', 's-1', 'Brunt frequency')
     call addfld('RELVAR',(/'lev'/), 'A', 'kg/kg', 'SHOC cloud liquid relative variance')
+    call addfld('ICE_CLOUD_FRAC',(/'lev'/), 'A', '', 'Ice number aware cloud fraction')
+    call addfld('PRECIP_ICE_FRAC',(/'lev'/), 'A', '', 'Precipitating ice fraction')
```

Comment:
I feel like precip_ice_frac will get misinterpreted a lot - is it a kind of precipitating ice mass? I'd like to rename it, but I struggle a bit with what a better name would be. Perhaps "PRECIPITATING_ICE_FRAC"?